docs/release-notes.txt

```diff
@@ -3,6 +3,134 @@
+  - For PBT, ``searcher.steps_per_round`` has been replaced by ``searcher.length_per_round``.
+  - For all experiments, the names for ``min_validation_period`` and ``min_checkpoint_period`` are unchanged but they are now configured in terms of records, batches or epochs.
+
+- **WebUI changes**
+
+  - Tasks List: ``/det/tasks``
+
+    - Consolidate notebooks, tensorboards, shells, commands into single list page.
+    - Add type filter to control which task types to display. By default all task types are shown when none of the types are selected.
+    - Add type column with iconography to train users to familiarize task types with visual indicators.
+    - Convert State filter from multi-select to single-select.
+    - Convert actions from expanded buttons to overflow menu (triple vertical dots).
+    - Move notebook launch buttons to task list from notebook list page.
+    - Add pagination support that auto turns on when entries extend beyond 10 entries.
+    - Add list of TensorBoard sources in a table Source column.
+
+  - Experiment List: ``/det/experiments``
+
+    - State filter converted from multi-select to single-select.
+    - Convert actions from expanded buttons to overflow menu (triple vertical dots).
+    - Batch operation logic change to available if the action can be applied to any of the selected experiments
+    - Add pagination support that auto turns on when entries extend beyond 10 entries.
+
+  - Experiment Detail: ``/det/experiments/<id>``
+
+    - Implement charting with plotly with zooming capability.
+    - Trial table paginates on the webui side in preparation for API pagination in the near future.
+    - Convert steps to batches in trials table and metric chart.
+    - Update continue trial flow to use batches, epochs or records.
+    - Use Monaco editor for the experiment config with YAML syntax highlighting.
+    - Add links to source for Checkpoint modal view, allowing users to navigate to the corresponding experiment or trial for the checkpoint.
+
+  - Trial Detail: ``/det/trials/<id>``
+
+    - Add trial information table.
+    - Add trial metrics chart.
+    - Implement charting with plotly with zooming capability.
+    - Trial info table paginates on the webui side in preparation for API pagination in the near future.
+    - Add support for batches, records and epochs for experiment config.
+    - Convert metric chart to show batches.
+    - Convert steps table to batches table.
+
+  - Master Logs: ``/det/logs``, Trial Logs: ``/det/trials/<id>/logs``, Task Logs: ``/det/<tasktype>/<id>/logs``
+
+    - Logs now stream in new logs events on the fly.
+    - Limit logs to 1000 lines for initial load and load additional 1000 for each subsequent older log fetches.
+    - Use new log viewer optimized for efficient rendering.
+    - Introduce log line numbers.
+    - Add ANSI color support.
+    - Add error, warning, and debug visual icons and colors.
+    - Add tailing button to enable tailing lob behavior.
```

Comment:
I don't think these are new to this release cc @hkang1

Reply:
@hamidzr is right. The logs changes were previously applied to Master logs and trial logs, but now they have been applied to task logs. I don't think it hurts have them here again due to the nature of the release.